2019-04-09_protocol.md

```diff
@@ -79,7 +79,26 @@ Aaron thinks parsimonie over time works like that:
 Aaron concludes that theory B' is not more corroberated then theory A. Becouse here theorie B' is theory B' in disguise. It follows that the parsimonie of theory B, therefore 10, applies and only for the future, parsimonie of B' (9) applies. However in this example A & B set out equally parsimonious. It is a truism that implicit, unformalised theories are less parsimonious then there formal conterpart. It can be argued that researchers theory B, before looking at the data is of the former kind and therefore less parsimonious than theory A. These commonly found circumstances are what justyfies the parsimonie 'bonus' for theory A.
 
 
-#### Reading suggestions
+#### Max's Answer
+
+"* Researcher B looks at the results and changes his Theory; based on a effect he had expected, but was not supported by data, he removes the effect from theory B creating B'
+* Researchers theory B' has now a apparently parsimonie of 9" - *One problem here is that you again set theory and hypothesis to be equal, which only works for the weekest of all theories, that have as many sentences as hypotheses. However, to illustrate your example: if the 10 parameters are 10 proposed causal effects, and theory A says all 10 exist, Theory B' (proposed after the fact) says 9 of them exist, I would agree that theory B' is more parsimonious. You may want to object "But aren't we just incorporating noise in our theory then?" - maybe we are, maybe not, there is a chance this is noise we sample. But the important thing is: **the sampling process does not care about the point of time we stated our theory!** Why should it be more probable that the 0 effect proposed after the fact is noise than that one of the other 9 non-zero effects proposed beforehand are noise?*
+
+I think the example you give Aaron is, again, one of a very week theory - 10 parameters, 10 hypotheses and 10 sentences in the theory - to be honest, this is not really a theory at all, because it entails all "Beobachtungssätze". No abstraction at all! Of course, often an easy example is a good starting point, but in this case I would argue it is almost pointless to argue about the parsimony of those "theories". Try to give a more complex example, and you will see, that your approach does not work anymore. Why? Because:
```

Comment:
That is not true in the literal sense, if you observe 10 numerical values, you get a lot more then 10 "Beobachtungssätze". If we only restrict our self to the covariance matrix we get 55 unique variables which may be all explained by ten free parameters of g-factor sem. Which was the case I had in mind while constructing the example.